pulsar-common/src/main/java/org/apache/pulsar/common/util/netty/EventLoopUtil.java

```diff
@@ -65,8 +65,8 @@ public static EventLoopGroup newEventLoopGroup(int nThreads, boolean enableBusyW
                     try {
                         CpuAffinity.acquireCore();
                     } catch (Throwable t) {
-                        log.warn("Failed to acquire CPU core for thread {}", Thread.currentThread().getName(),
-                                t.getMessage(), t);
+                        log.warn("Failed to acquire CPU core for thread {}, the error message is {}",
```

Comment:
log.warn will print the stacktrace when you pass a Throwable argument, so no need for "the error message is {}"

Reply:
I already fix it. :)